pkg/middleware/middleware.go

```diff
@@ -49,6 +49,7 @@ func GetContextHandler() macaron.Handler {
 			initContextWithAuthProxy(ctx, orgId) ||
 			initContextWithUserSessionCookie(ctx, orgId) ||
 			initContextWithAnonymousUser(ctx) {
+			log.Debug2("Context initialized")
```

Comment:
This logline is too verbose. I think we should rewrite the chain rather than add dummy logging.